platform/darwin/src/MGLOfflineStorage.h

```diff
@@ -341,6 +355,71 @@ MGL_EXPORT
  */
 @property (nonatomic, readonly) unsigned long long countOfBytesCompleted;
 
+
+#pragma mark - Managing Ambient Cache
+
+/**
+ Sets the maximum ambient cache size in megabytes. The default maximum cache
+ size is 50 MB. To disable ambient caching, set the maximum ambient cache size
+ to `0`. Setting the maximum ambient cache size does not impact the maximum size
+ of offline packs.
+ 
+ While this method does not limit the space available to offline packs,
+ data in offline packs count towards this limit. If the maximum ambient
+ cache size is set to 30 MB and 20 MB of offline packs are downloaded,
+ there may be only 10 MB reserved for the ambient cache.
+ 
+ This method should be called before the map and map style have been loaded.
+ 
+ This method is potentially expensive, as the database will trim cached data
```

Comment:
Since there are several comments/questions about things that match the core and Android docs, should this PR be blocked from going into the beta? Can I address any docs issues specific to this PR now so I can merge today, then we can discuss the issues that are cross-platform later?

Reply:
@jmkiley considering that the core and Android implementation have already landed, and you're being consistent with them, I agree we should not block this PR on those grounds and address those concerns across core, Android, and iOS as a separate PR (before the release).

Reply:
Yeah I think this is a good idea.